homeassistant/components/sensor/eliqonline.py

```diff
@@ -54,11 +54,6 @@ def name(self):
         return self._name
 
     @property
-    def icon(self):
```

Comment:
I've already merged your other PR to add this icon so now this PR is removing that icon again…